application/views/view_log/qso.php

```diff
@@ -443,6 +443,13 @@
                     </tr>
                     <?php } ?>
 
+                    <?php if($row->COL_EMAIL != null) { ?>
+                    <tr>
+                        <td><?= __("eMail"); ?></td>
+                        <td><a href="mailto: "<?php echo $row->COL_EMAIL.'">'.$row->COL_EMAIL.'</a>'; ?></td>
```

Comment:
little UI bug

```suggestion
                        <td><a href="mailto:<?php echo $row->COL_EMAIL; ?>"><?php echo $row->COL_EMAIL; ?></a></td>
```

<img width="303" alt="image" src="https://github.com/user-attachments/assets/027b92d4-6061-423c-a032-c06a29f6c7d3">
<img width="180" alt="image" src="https://github.com/user-attachments/assets/ee28fad8-076a-483c-bbb9-73ba9225b414">